source/layouts/partials/_head.html.erb

```diff
@@ -40,9 +40,6 @@
   <link rel="preload" href="<%= asset_url("fonts/opensans-light-webfont-webfont.woff") %>" as="font" type="font/woff2" crossorigin>
   <link rel="preload" href="<%= asset_url("fonts/opensans-regular-webfont-webfont.woff") %>" as="font" type="font/woff2" crossorigin>
 
-  <!-- Preload some Javascript files we will load in the future. -->
```

Comment:
If you do see Chrome say there's a preload available, add it back in 😉…

I have no idea what Embedly does..